### PR TITLE
Use feature gate for originating identity.

### PR DESCRIPTION
**Pull request by staebler**

### CODE REVIEW

File: pkg/features/kube_features.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	// owner: @pmorie
+	// alpha: v1.7
+	//
+	// OriginatingIdentity controls whether the controller should include originating identity in the header of requests
+	// sent to brokers
+	OriginatingIdentity utilfeature.Feature = "OriginatingIdentity"
+)
+
+func init() {
+	utilfeature.DefaultFeatureGate.Add(defaultServiceCatalogFeatureGates)
```
Comment (deads2k) [inline pkg/features/kube_features.go]: At some point, we should start encouraging people to create their own `FeatureGate`.  I don't know for sure that everything wires through, but if it does, do that here.

**Comment by deads2k** [comment]
I'd encourage using a separate instance and I missed where you were adding the feature to the flags.

File: pkg/features/kube_features.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/*
```
Comment (pmorie) [inline pkg/features/kube_features.go]: Why is this called `kube_features.go`?

**Review by pmorie — COMMENTED** [review]
One question on the name of `kube_features.go` and I will LGTM.

**Pull request closed without merging**